soroban-spec/src/gen/typescript/project_template/tsconfig.json

```diff
@@ -95,4 +95,4 @@
   "include": [
     "src/*"
   ]
-}
\ No newline at end of file
+}
```

Comment:
@chadoh @willemneal I've noticed you folks are flipping file endings. I think one of you, maybe @willemneal, might have new-lines at end of files turned off. Would be great to fix that in your editor to avoid permadiffs.

Reply:
Thanks for the note; will check